### PR TITLE
Feature/cleanup config

### PR DESCRIPTION
**Pull request by insacuri**
## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,6 @@ The main configuration options are listed below:
 | SEFT_RABBITMQ_PORT                    | '5672'                         | Port for rabbit mq 1
 | SEFT_RABBITMQ_PORT2                   | '5672'                         | Port for rabbit mq 2
 | SEFT_CONSUMER_RABBIT_QUEUE            | `Seft.Responses`               | Incoming queue to read from
-| SEFT_RABBIT_EXCHANGE                  | `message`                      | RabbitMQ exchange to use
 | SEFT_CONSUMER_RABBIT_QUARANTINE_QUEUE | `"Seft.Responses.Quarantine"`  | Rabbit quarantine queue
```
Comment (gemmai95) [inline README.md]: Does this quarantine queue need removed from readme as well?

File: README.md [hunk 2]
```diff
@@ -44,7 +44,6 @@ The main configuration options are listed below:
 | SEFT_RABBITMQ_PORT                    | '5672'                         | Port for rabbit mq 1
 | SEFT_RABBITMQ_PORT2                   | '5672'                         | Port for rabbit mq 2
 | SEFT_CONSUMER_RABBIT_QUEUE            | `Seft.Responses`               | Incoming queue to read from
```
Comment (gemmai95) [inline README.md]: And queue as these are no longer configurable by environment variables?

Comment (insacuri) [reply README.md]: Agreed!  Removed now

File: README.md (hunk 1 above)
Comment (insacuri) [reply README.md]: Removed

**Comment by insacuri** [comment]
Don't merge until log location in prod has been changed!

**Pull request merged**